src/lib/booleans.js

```diff
@@ -0,0 +1,77 @@
+function negate(a) {
+  return !a;
+}
+
+function both(a, b) {
+  return a && b;
+}
+
+function either(a, b) {
+  return a || b;
+}
+
+function none(a, b) {
+  return !a && !b;
+}
+
+function one(a, b) {
+  return (a && b === false) || (b && a === false);
```

Comment:
This doesn't seem right. Should this function return true if ONLY one of a or b is true?
For example: 

```js
one(true, false)   // returns true
one(false, true)   // returns true
one(true, true)    // returns false
one(false, false)  // returns false
```